docs/layouts/_default/baseof.html

```diff
@@ -23,5 +23,143 @@
+                  role: 'accept_all',
+                },
+                secondary_btn: {
+                  text: 'Reject all',
+                  role: 'accept_necessary',
+                }
+              },
+              settings_modal: {
+                title: 'Cookie preferences',
+                save_settings_btn: 'Save settings',
+                accept_all_btn: 'Accept all',
+                reject_all_btn: 'Reject all',
+                close_btn_label: 'Close',
+                cookie_table_headers: [
+                  { col1: 'Name' },
+                  { col2: 'Domain' },
+                  { col3: 'Expiration' },
+                  { col4: 'Description' },
+                ],
+                blocks: [
+                  {
+                    title: 'Cookie usage 📢',
+                    description: 'Cookies are used to ensure the basic functionalities of the website and to enhance your online experience. You can choose for each category to opt-in/out whenever you want. For more details relative to cookies and other sensitive data, please read the full <a href="https://coherentlabs.github.io/GameUIComponents/privacy-policy/" class="cc-link">privacy policy</a>.'
+                  }, {
+                    title: 'Strictly necessary cookies',
+                    description: 'These cookies are essential for the proper functioning of my website. Without these cookies, the website may not work properly.',
+                    toggle: {
+                      value: 'necessary',
+                      enabled: true,
+                      readonly: true,
+                    }
+                  }, {
+                    title: 'Performance and Analytics cookies',
+                    description: 'These cookies allow the website to remember the choices you have made in the past.',
+                    toggle: {
+                      value: 'analytics',
+                      enabled: false,
+                      readonly: false,
+                    },
+                    cookie_table: [
+                      {
+                        col1: '^_ga',
+                        col2: 'google.com',
+                        col3: '2 years',
+                        col4: 'This cookie is a Google Analytics persistent cookie.',
+                        is_regex: true,
+                      },
+                      {
+                        col1: '_gid',
+                        col2: 'google.com',
+                        col3: '1 day',
+                        col4: 'This cookie is a Google Analytics persistent cookie.',
+                      }
+                    ]
+                  }
+                ]
+              }
+            }
+          }
+        });
+
+        // Toggle dark mode
+        const htmlElement = document.getElementsByTagName('html')[0];
+
+        if (htmlElement.hasAttribute('data-dark-mode')) {
+          document.body.classList.add('c_darkmode');
+        }
```

Comment:
Can we set the style through a CSS variable that is changed according to the theme? Now if you switch the theme after the popup has been shown it does not get updated with the correct theme. Using a var will fix this.